pt/lessons/ecto/associations.md

```diff
@@ -0,0 +1,505 @@
+---
+version: 1.0.0
+title: Associações
+---
+
+Nessa seção vamos aprender a utilizar o Ecto para definir e trabalhar com associações entre esquemas.
+
+{% include toc.html %}
+
+## Configuração
+
+Nós vamos construir a aplicação `Example`, das ultimas lições. Você pode referir-se a configuração [aqui](./basics.md) para
+uma breve recapitulação.
+
+## Tipos de Associações
+
+Existem três tipos de associações que podem ser definidas entre nossos esquemas. Vamos dar atenção ao que elas são e como
+impementar cada um dos tipos.
+
+### Belongs To/Has Many
+
+Nós estamos adicionando algumas novas entidades ao modelo de domínio da aplicação de exemplo para que seja possível categorizar
```

Comment:
@lucaspbordignon could you make this paragraph a single-line, to follow the same format that the english use? So in future updates to the translation we can just follow the line numbers to see what changed.